### PR TITLE
[RUN-318] Changed spacing between header and rune

### PR DESCRIPTION
**Pull request by Kengii**
## Description 📝

Changed spacing between header and rune, added constants.

---

### Test ✅

Simulator: SE, SE 3 gen, 8 plus, X, 14, 13 mini, 14 pro max

### CODE REVIEW

File: Runar/Runar/Controllers/Library flow/LibraryCells/LibraryRuneCell.swift [hunk 1]
```diff
@@ -122,3 +121,25 @@ public class LibraryRuneCell: LibraryCell {
...
+    static let runeTitleFontSize = 24.0
+    
+    static let runeImageHeight = 129.0
+    static let runeImageTopAnchor = 10.0
+    static let runeImageSize = CGSize(width: 109, height: 129)
+    
+    static let tagsCVTopAnchor = 10.0
+    
+    static let runeDescTopAnchor = 10.0
+    static let runeDescBottomAnchor = -26.0
```
Comment (SkrigalovskyAndrey) [inline Runar/Runar/Controllers/Library flow/LibraryCells/LibraryRuneCell.swift]: Avoid use negative numbers

**Review by arti1233 — APPROVED** [review]

**Review by Qeezii — APPROVED** [review]

File: Runar/Runar/Controllers/Library flow/LibraryCells/LibraryRuneCell.swift [hunk 2]
```diff
@@ -122,3 +117,26 @@ public class LibraryRuneCell: LibraryCell {
...
     }
 }
+
+private extension CGFloat {
+    static let runTitleTopAnchor = 15.0
+    static let runeTitleHeight = 26.0
+    static let runeTitleFontSize = 24.0
+    
+    static let runeImageTopAnchor = 10.0
+    static let runeImageSize = CGSize(width: 109, height: 129)
```
Comment (AliaksandrNiehaj) [inline Runar/Runar/Controllers/Library flow/LibraryCells/LibraryRuneCell.swift]: Why do you use static size?

File: Runar/Runar/Controllers/Library flow/LibraryCells/LibraryRuneCell.swift [hunk 3]
```diff
@@ -25,45 +25,41 @@ public class LibraryRuneCell: LibraryCell {
         self.separatorInset = UIEdgeInsets(top: 0, left: 26, bottom: 0, right: 26)
         
         runeTitle.translatesAutoresizingMaskIntoConstraints = false
-        NSLayoutConstraint.activate([
-            runeTitle.topAnchor.constraint(equalTo: topAnchor, constant: 26),
-            runeTitle.heightAnchor.constraint(equalToConstant: 26),
-            runeTitle.centerXAnchor.constraint(equalTo: centerXAnchor)
-        ])
+        runeTitle.snp.makeConstraints { make in
+            make.top.equalTo(safeAreaLayoutGuide).offset(CGFloat.runTitleTopAnchor)
```
Comment (AliaksandrNiehaj) [inline Runar/Runar/Controllers/Library flow/LibraryCells/LibraryRuneCell.swift]: Explain, please, why safeAreaLayoutGuide?

File: Runar/Runar/Controllers/Library flow/LibraryCells/LibraryRuneCell.swift [hunk 4]
```diff
@@ -25,45 +25,41 @@ public class LibraryRuneCell: LibraryCell {
...
+            make.height.equalTo(CGFloat.runeTitleHeight)
```
Comment (AliaksandrNiehaj) [inline Runar/Runar/Controllers/Library flow/LibraryCells/LibraryRuneCell.swift:30]: You should avoid using types while using static var/let/func

File: Runar/Runar/Controllers/Library flow/LibraryCells/LibraryRuneCell.swift [hunk 5]
```diff
@@ -122,3 +117,26 @@ public class LibraryRuneCell: LibraryCell {
...
+
+private extension CGFloat {
+    static let runTitleTopAnchor = 15.0
+    static let runeTitleHeight = 26.0
+    static let runeTitleFontSize = 24.0
+    
+    static let runeImageTopAnchor = 10.0
+    static let runeImageSize = CGSize(width: 109, height: 129)
+    
+    static let tagsCVTopAnchor = 16.0
```
Comment (AliaksandrNiehaj) [inline Runar/Runar/Controllers/Library flow/LibraryCells/LibraryRuneCell.swift]: Please, explain, what is CV?

File: Runar/Runar/Controllers/Library flow/LibraryCells/LibraryRuneCell.swift [hunk 6]
```diff
@@ -122,3 +117,26 @@ public class LibraryRuneCell: LibraryCell {
...
+    
+    static let runeDescTopAnchorWithTagsCV = 74.0
+    static let runeDescTopAnchorWithOutTagsCV = 16.0
+    static let runeDescTrailigLeadingAnchor = 24.0
+    static let runeDescBottomAnchor = 34.0
+    static let runeDescFontSize = DeviceType.iPhoneSE || DeviceType.isIPhone678 ? 16.0 : 19.0
+}
+
+private extension UIColor {
+    static let runeDescColor = UIColor(red: 0.855, green: 0.855, blue: 0.855, alpha: 1)
```
Comment (AliaksandrNiehaj) [inline Runar/Runar/Controllers/Library flow/LibraryCells/LibraryRuneCell.swift]: Could you, please, ask UI/UX team about this color?

**Review by AliaksandrNiehaj — CHANGES_REQUESTED** [review]

File: Runar/Runar/Controllers/Library flow/LibraryCells/LibraryRuneCell.swift (hunk 5 above)
Comment (Kengii) [reply Runar/Runar/Controllers/Library flow/LibraryCells/LibraryRuneCell.swift]: Tags CollectionView

**Review by SkrigalovskyAndrey — APPROVED** [review]

**Review by AliaksandrNiehaj — APPROVED** [review]

**Review by Qeezii — APPROVED** [review]

**Review by SkrigalovskyAndrey — APPROVED** [review]

**Review by FilyutaDima — APPROVED** [review]

**Comment by Kengii** [comment]
before: top anchor 26 
Снимок экрана 2023-03-04 в 23 08 38
after: top anchor as in figma 15
Снимок экрана 2023-03-04 в 23 09 06

**Pull request merged**